scripts/proguard.pro

```diff
@@ -3,6 +3,7 @@
 
 -optimizationpasses 5
 -verbose
+-ignorewarnings #until https://github.com/wvengen/proguard-maven-plugin/pull/76
```

Comment:
This makes me nervous, because a lot of the effort in setting up the proguard plugin correctly is making it so that it doesn't throw warnings on library versions. That link seems to be about Java 9... are we making java 9 binaries now? If so, why?

Reply:
this is because some of the required libraries have `meta-inf/versions/9` folders that proguard tries to use, finds a class path conflict and warns/crashes on

Reply:
ok so this is fixed with java 16 when on my 1.17 fork using modifications to these build scripts

Reply:
figured out that one of the other things fixed it somehow. not sure how but it doesn't throw errors now...